corehq/apps/domain/views/base.py

```diff
@@ -65,6 +69,22 @@ def select(request, do_not_redirect=False, next_view=None):
         return render(request, domain_select_template, additional_context)
 
 
+Link = namedtuple('Link', ('name', 'url'))
+
+
+@quickcache(['couch_user.username'])
+def get_domain_dropdown_links(couch_user, view_name="domain_homepage"):
+    domain_objects = set(Domain.active_for_user(couch_user))
```

Comment:
why is this a `set`

Reply:
It's left over from an earlier iteration. Reworked this code in 0d7c26819078320dbe8f11d6cae15282270a8d81